tests/fixture/tmpnet/genesis.go

```diff
@@ -148,9 +148,7 @@ func NewTestGenesis(
 
 	// Define C-Chain genesis
 	cChainGenesis := &core.Genesis{
-		Config: &params.ChainConfig{
-			ChainID: big.NewInt(43112), // Arbitrary chain ID is arbitrary
-		},
+		Config:     params.AvalancheLocalChainConfig,
```

Comment:
tbh it's a little awkward that avalanchego takes dependencies on `coreth/params` and  and `coreth/core` in this file.
In the long run coreth should probably provide a better method to create the genesis for external users.